resources/bin/ballerina

```diff
@@ -21,6 +21,15 @@ else
     CURRENT_PATH="$(dirname "$(readlink -f "$0")")"
 fi
 
+JAVA_COMMAND=java
+if test -d "$CURRENT_PATH/../dependencies/jdk-11.0.8+10-jre"; then
+  JAVA_COMMAND="$CURRENT_PATH/../dependencies/jdk-11.0.8+10-jre/bin/java"
+else if test -d "$CURRENT_PATH/../dependencies/jdk8u265-b01-jre"; then
+  JAVA_COMMAND="$CURRENT_PATH/../dependencies/jdk8u265-b01-jre/bin/java"
+else if test -d "$CURRENT_PATH/../dependencies/jdk8u202-b08-jre"; then
+  JAVA_COMMAND="$CURRENT_PATH/../dependencies/jdk8u202-b08-jre/bin/java"
+fi
+
```

Comment:
If all three JDK dependencies are there, which one will be picked? Will it pick the JDK which is compatible with the currently active ballerina distribution?

Reply:
We may pack one with installers but user may have any of above if they download from `ballerina update` command